app/auto-eligibility/config/index.js

```diff
@@ -0,0 +1,26 @@
+const Joi = require('joi')
+
+const schema = Joi.object({
+  emailTemplates: {
+    waitingList: Joi.string().uuid(),
+    genericIneligible: Joi.string().uuid(),
+    applyServiceInvite: Joi.string().uuid()
+  }
+})
+
+const config = {
+  emailTemplates: {
```

Comment:
A better name maybe: "emailTemplateIds"?

Reply:
Sure, bit clearer they are identifiers for external templates this way!